opm/material/densead/EvaluationFormat.hpp

```diff
@@ -0,0 +1,57 @@
+// -*- mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
+// vi: set et ts=4 sw=4 sts=4:
+/*
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 2 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+
+  Consult the COPYING file in the top-level source directory of this
+  module for the precise wording of the license and the list of
+  copyright holders.
+*/
+
+#ifndef OPM_EVALUATION_FORMAT_HPP
+#define OPM_EVALUATION_FORMAT_HPP
+
+#include <opm/material/densead/Evaluation.hpp>
+
+#include <fmt/format.h>
+#include <fmt/ranges.h>
+
```

Comment:
If this header is installed then this breaks for people that install opm-common but use the bundled fmt from there.

Reply:
what breaks? 

if they try to use the header, yes, but this is not (supposed to be) included anywhere libfmt is not available. in that case they'll get a compiler error.